CHANGELOG.md

```diff
@@ -3,6 +3,56 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 5.13.0
+
+_Jul 7, 2022_
+
+We'd like to offer a big thanks to the 9 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 `@mui/x-date-pickers` and `@mui/x-date-pickers-pro` are moving to beta
```

Comment:
@joserodolfofreitas not sure how to present that change

Reply:
```suggestion
- 🎁 `@mui/x-date-pickers` and `@mui/x-date-pickers-pro` are now in beta!
```
?